Assets/scss/components/_home-quote.scss

```diff
@@ -0,0 +1,192 @@
+.component-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    font-family: $openSans;
+
+    @include max-screen(lg) {
+        flex-direction: column;
+    }
+
+    .text-area {
+        width: 33%;
+        padding: toRem(96);
+
+        h1 {
+            font-size: toRem(32);
+            font-weight: bold;
+            margin-bottom: toRem(20);
+        }
+
+        p {
+            font-size: toRem(16);
+            line-height: 1.5;
+        }
+
+        @include max-screen(lg) {
+            width: 100%;
+            padding: toRem(20);
+        }
+    }
+
+    .image-area {
+        width: 67%;
```

Comment:
Instead of using widths on each of these elements, with media queries overriding for smaller to 100%, you could use grid on the container and set `grid-template-columns`. This isn't drastically different, just controlled from the parent instead of each child needing their own widths/overrides.